Update metadata-keys.adoc (#152)
mulesoft/docs-mule-sdk: modules/ROOT/pages/metadata-keys.adoc

// _TODO Fix broken Image
// image::<missing>metadata/keys_single_drop.png[MetadataKeys Single Level Dropdown]

> After the resolver is defined, you can use it as an operation or source by adding a `@MetadataScope` annotation with its `keysResolver` value at the class definition level and also annotating the parameter that is the key with `@MetadataKeyId`. All metadata resolvers must be of the same category in an operation or source.

Reviewer: ```suggestion
After the resolver is defined, you can use it as an operation or a source by adding the `@MetadataScope` annotation with its `keysResolver` value at the class definition level and annotating the parameter that is the key with `@MetadataKeyId`. All metadata resolvers must belong to the same category in an operation or a source.
```
